libraries/AP_Mount/SoloGimbal_Parameters.cpp

```diff
@@ -126,7 +128,7 @@ void SoloGimbal_Parameters::update()
 
     // retry initial param retrieval
     if(!received_all()){
-        if (tnow_ms-_last_request_ms > _retry_period) {
+        if ((tnow_ms-_last_request_ms > _retry_period) && (HAVE_PAYLOAD_SPACE(_chan, PARAM_REQUEST_LIST_SEND))) {
```

Comment:
compile is failing. To fix, change:
`PARAM_REQUEST_LIST_SEND`
to
`PARAM_REQUEST_LIST`